db/migrate/20180810144738_update_default_internal_attribute.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateDefaultInternalAttribute < ActiveRecord::Migration[5.0]
+  class ServiceTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+
+    include MigrationStubHelper
+  end
+
+  def up
+    change_column :service_templates, :internal, :boolean
+    say_with_time("Set ServiceTemplate internal") do
+      ServiceTemplate.where.not(:type => "ServiceTemplateTransformationPlan").update_all(:internal => false)
+    end
+  end
+
+  def down
```

Comment:
We don't need `down`. It was a bug. We should set the column to `nil` at all.

Reply:
+1 I don't think this migration should have a `down` because we don't want to put the data back into a bad state.